ast/dml.go

```diff
@@ -1143,6 +1144,9 @@ func (n *LoadDataStmt) Restore(ctx *RestoreCtx) error {
 	}
 	ctx.WriteKeyWord("INFILE ")
 	ctx.WriteString(n.Path)
+	if n.OnDuplicate == OnDuplicateKeyHandlingReplace {
+		ctx.WriteKeyWord(" REPLACE")
+	}
```

Comment:
Without `LOCAL`, `LOAD DATA … IGNORE INTO` and `LOAD DATA … INTO` are different and should not be omitted.

Since `LOAD DATA LOCAL … IGNORE INTO` and `LOAD DATA LOCAL … INTO` are equivalent, I suggest always restore as `LOAD DATA LOCAL … IGNORE INTO` to simplify the code structure.

Reply:
yeah, it makes sense.

Reply:
fixed